### ISSUE TITLE
Drag center issue when zoomed

### ISSUE THREAD

**Issue by gaearon** [gaearon/react-dnd-html5-backend#12]
Moved from https://github.com/gaearon/react-dnd/issues/180.
Original issue by @adjohu.

---

If I zoom out and start dragging elements, they appear offset from the mouse.

**Comment by gaearon** [comment]
Yep, that's a bug. I'd appreciate somebody looking into this. Some coordinates in [OffsetUtils](https://github.com/gaearon/react-dnd-html5-backend/blob/master/src/OffsetUtils.js) seem to be calculated incorrectly on zoom.

**Comment by ColemanGariety** [comment]
can we re-open this? I'm curious about the pull req

**Comment by gaearon** [comment]
The issue is open.

**Comment by iamsolankiamit** [comment]
Is this issue resolved?

**Comment by fortinmike** [comment]
Also having issues with this. I'm very interested in a solution!
